.github/workflows/prepare-release.yml

```diff
@@ -156,25 +149,20 @@ jobs:
       - name: Temporary tag
         run: git tag "$INPUT_TAG"
 
-      - name: Set up Java
-        # FIXME: v4+ requires Node 20
-        uses: actions/setup-java@v3
-        with:
-          java-version: "11"
-          distribution: "temurin"
-
-      - name: Set up Gradle
-        # FIXME: v3+ requires Node 20
-        uses: gradle/gradle-build-action@v2
-      
       - name: Build native test server
-        run: ./gradlew :temporal-test-server:build
+        run: |
```

Comment:
Is there a reason you didn't run this task in a `container`?

Reply:
Wouldn't this step still make sense to run in a container?

Reply:
The `container` parameter only exists at the job-level. Steps run with whatever runner/container was set by the job.

A possible alternative is to use a docker image as an action, but there are a few limitations to that approach. But trying it that now to see if that would work.